### ISSUE TITLE
Необходимо сделать сайт адаптивным под мобильные устройства.

### ISSUE THREAD

**Issue by semen1032** [BearPro/seterator#11]
Требуется переработать верстку с упором на экраны небольшого размера.
Следует добавить медиа-запросы, переработать ширину блоков, размер картинок, а также размер текста.

**Comment by BearPro** [comment]
Не проще просто поддерживать дубликат каждой въюшки с теми же стилями, но свёрстанный под портретный экран?

**Comment by semen1032** [comment]
Нет. Это будет сложнее - дублирование кода + при добавлении новых фич нужно опять же прописывать их в обе вьюшки. Тем более это старый подход, а мы используем современные технологии, которые позволяют без особых усилий адаптировать сайт. Также можно будет оптимизировать сайт и под различные "альбомные" разрешения.

**Comment by BearPro** [comment]
Ну а как это будет выглядеть? Представление же должно действительно сильно меняться при изменении разрешения. Элементарные кнопки, из которых строиться дизайн, как я понимаю, одинаковые во всех версиях, а вот их расположение наверняка будет отличаться сильнее, чем можно сделать одной только голой вёрсткой.

А делать адаптивность на js без фреймворка - тоже глухая затея, мы утонем в собственном быдлокоде.

Может быть рассмотреть на тег-хелперы в Razor? Чтобы определить в них все базовые элементы дизайна, и тогда мобильная и дестктопная версия будут отличаться только параметрами и порядком вызова их.

**Comment by semen1032** [comment]
Дизайн не будет меняться кардинально. Последовательность расположения блоков в html останется такой же.
При уменьшении ширины экрана, блоки, следующие друг за другом в горизонтальном направлении будут съезжать вниз. Допустим, у десктопа на главной 9 компетишинов, расположенные в 3 строчки по 3 колонки. В мобилке они могут перераспределиться в 9 строчек по 1 колонке. 
Хедер также будет не горизонтальным, а вертикальным + он будет скрываться в кнопку, при нажатии на которую можно увидеть сам хедер. 
Вот пример из гугла: https://grid-cats.glitch.me/
И это только один из вариантов. Также в некоторых вариантах можно применять display: none на некоторые элементы в зависимости от разрешения (как, например, в кнопке с хедером). 
Я думаю, существует еще множество фишек для адаптации под мобилки. Так что не вижу тут большой проблемы.

**Comment by semen1032** [comment]
Страницы, которые требуется обновить под мобильные устройства:
- [ ]  Главная
- [ ]  Страница конкурса
- [ ]  Страница создания конкурса
- [ ]  Страница профиля
- [ ]  Страница подачи заявки
- [ ]  Страница регистрации/авторизации
- [ ]  Хедер
- [ ]  Футер
